rustfmt.toml

```diff
@@ -1,2 +1,3 @@
 use_field_init_shorthand = true
-edition = "2021"
+edition = "2024"
```

Comment:
I think this could cause issues because the project code itself is not actually edition 2024 yet.
I think its best to remove this line (which will result in `edition = "2021"` being automatically selected)
And then keeping the below line.